xml/operating_system_darwin.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Patterns for common names of Mac OS X operating systems.
+-->
+<fingerprints matches="operating_system.name">
+  <!-- Match Mac OS Classic first due to weak matching on Mac OS X -->
+  <fingerprint pattern="^(?i:(?:Apple )?Mac OS ([7-9](?:\.\d+?)*?))$">
```

Comment:
Apple OS 10.x still looks pretty weird to me. But maybe there are such banners out there...